### PR TITLE
foundations- Global Styles(created folder)

### PR DESCRIPTION
**Pull request by Theweeksad**
added global styles folder.

### CODE REVIEW

**Comment by francoo-trainer** [comment]
NICE BRO!

**Review by Jasoneto — APPROVED** [review]
Nice. I found the Suggested Changes.

File: index.html [hunk 1]
```diff
@@ -6,8 +6,9 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Home</title>
 
-    <link rel="stylesheet" href="./main.css">
-    <link rel="stylesheet" href="./Global-styles.css">
+    <link rel="stylesheet" href="./CSS/Global-styles.css">
```
Comment (Jasoneto) [inline index.html:9]: I remember Johan told us that typography go first, but the code is perfect. Nice job Jefry Mauricio.

**Review by Jasoneto — APPROVED** [review]
The code achieved the ticket requirements.

**Review by francoo-trainer — COMMENTED** [review]
good contribution. !

File: index.html (hunk 1 above)
Comment (Theweeksad) [reply index.html:9]: I'll make the changes

**Review by Brayan1425 — APPROVED** [review]
I think the changes are good

**Review by johanmendezb — APPROVED** [review]

**Review by francoo-trainer — APPROVED** [review]

**Pull request merged**